data.sql

```diff
@@ -3,3 +3,14 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_k
 ('2', 'Gabumon', 'Nov 15 2018', '2', 'Yes', '8'),
 ('3', 'Pikachu', 'Jan 7 2021', '1', 'No', '15.04'),
 ('4', 'Devimon', 'May 12 2017', '5', 'Yes', '11');
+
+-- Query and Update animals table
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('5', 'Charmander', 'Feb 8 2020', '0', 'No', '11'),
+('6', 'Plantmon', 'Nov 15 2021', '2', 'Yes', '5.7'),
+('7', 'Squirtle', 'Apr 2 1993', '3', 'No', '12.13'),
+('8', 'Angemon', 'Jun 12 2005', '1', 'Yes', '45'),
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix your `weight_kg` items symbol, these block items `weight_kg` items should be given with `minus(-)` 🙏🏼 

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```